### ISSUE TITLE
Pressing link on mobile in Live Preview isn’t opening Hover Editor.

### ISSUE THREAD

**Issue by daxmonson** [nothingislost/obsidian-hover-editor#250]
First off, this is in the top five essentials. I’ve looked for a buy me a coffee Etc link before and never found one, so I’ll ask for that as a feature request for you. 

One thing that would make the mobile experience with it 10x here would be if (in Live Preview mode), the open in Hover Editor option worked. I can reliably bring the Modal up, but clicking on that text does nothing. It works fine in Reading mode. Maybe this should work, but there’s a plugin conflict?

I’ve come up with workarounds like with Commander, but that requires highlighting the link which is sometime finicky with the way I structure links and text before them. Thanks for considering it.

**Comment by pjeby** [comment]
As I really only use desktop, I don't understand what you mean here by "option" and "modal".  Are you talking about a long-press on a link to bring up a context menu?  Are you saying there's an option on that menu but it doesn't work?  Or are you talking about the command in the command palette?

**Comment by daxmonson** [comment]
Long-pressing on a link within a note and that opening the context menu where there's the option "open in hover editor" isn't opening the link in Live preview mode. Tapping on the link and selecting "open in hover editor" works when in reading mode. Would really be smooth if that worked in both modes. This should be easier to understand:  
![IMG_1285](https://github.com/nothingislost/obsidian-hover-editor/assets/5855930/5039bf91-37d1-47cd-97ff-5d1b2ac865c0)

**Comment by pjeby** [comment]
I've never seen "Open in Float Preview" before.  I'm wondering if that's a mobile-only thing, or is it another plugin?  Also, does it work?

**Comment by writtenfool** [comment]
"Open in Float Preview" is provided bye the Floating Search plugin. 

Appreciate your Hover Editor. I use it everywhere, everyday. It has sped up my ability to flit around and edit content with zero friction. I especially like the ability to see properties.

Cheers!

**Comment by daxmonson** [comment]
That’s right, floating search is another plugin and it also doesn’t “Open in float Preview” when Obsidian is in Live Preview mode. 

I know how it is programming, no need to implement this if it’s just me requesting it. I’ve come up with a few workarounds, varying from an extra tap to three. Thanks for your work on it.

**Comment by pjeby** [comment]
That sounds like it's a common issue with Obsidian mobile, if another plugin that shows things on top doesn't work.  I personally don't use Obsidian on mobile so I'm not sure what the deal is.   If I had to guess it would be that on a device with no pointer, HE and FS are probably guessing wrong as to where to place the popup.  Except then the question is why it works in reading mode...  so yeah, it's a bit unclear to me what's going on.
